projects/storefrontlib/src/lib/checkout/components/payment-method/payment-method.component.spec.ts

```diff
@@ -1,31 +1,29 @@
 import { Component, Input } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
-import { Observable, of } from 'rxjs';
-
+import { ActivatedRoute } from '@angular/router';
 import {
   Address,
-  CartDataService,
   CheckoutService,
   GlobalMessageService,
   I18nTestingModule,
-  RoutingService,
   PaymentDetails,
-  UserService,
   RoutesConfig,
   RoutingConfigService,
+  RoutingService,
+  UserService,
 } from '@spartacus/core';
+import { Observable, of } from 'rxjs';
 import { Card } from '../../../../shared/components/card/card.component';
-import { PaymentMethodComponent } from './payment-method.component';
-
-import createSpy = jasmine.createSpy;
-import { ActivatedRoute } from '@angular/router';
+import { defaultStorefrontRoutesConfig } from '../../../ui/pages/default-routing-config';
+import { CheckoutConfigService } from '../../checkout-config.service';
 import {
   CheckoutStep,
   CheckoutStepType,
 } from '../../model/checkout-step.model';
-import { CheckoutConfigService } from '../../checkout-config.service';
-import { defaultStorefrontRoutesConfig } from '../../../ui/pages/default-routing-config';
+import { PaymentMethodComponent } from './payment-method.component';
+
+import createSpy = jasmine.createSpy;
 
 @Component({
   selector: 'cx-icon',
```

Comment:
line 59: remove the userid param from the mock function (Can't put review on it so I put it here :P)

Reply:
I agree.  Fixed.